### PR TITLE
Add a Codacy badge to README.md

### PR DESCRIPTION
**Pull request by codacy-badger**

### CODE REVIEW

**Comment by lgarest** [comment]
This one closes #36

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # CunadoBot
 
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/0294cc92c1b04071913414a4f6d69f8d)](https://app.codacy.com/app/lgarest/cunadobot?utm_source=github.com&utm_medium=referral&utm_content=pablobuenaposada/cunadobot&utm_campaign=Badge_Grade_Dashboard)
 [![Build Status](https://travis-ci.org/pablobuenaposada/cunadobot.svg?branch=master)](https://travis-ci.org/pablobuenaposada/cunadobot)
```
Comment (semicolom) [inline README.md]: I think Build Status should go first.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # CunadoBot
 
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/0294cc92c1b04071913414a4f6d69f8d)](https://app.codacy.com/app/lgarest/cunadobot?utm_source=github.com&utm_medium=referral&utm_content=pablobuenaposada/cunadobot&utm_campaign=Badge_Grade_Dashboard)
```
Comment (semicolom) [inline README.md]: All these URL parameters are really needed?

File: README.md (hunk 1 above)
Comment (lgarest) [reply README.md]: It was created automatically 🤷‍♂️

**Pull request merged**